spec/src/main/asciidoc/method-query.asciidoc

```diff
@@ -352,6 +399,18 @@ The return type of a Query by Method Name is determined as indicated in the foll
 | `find` accepting a `PageRequest` | `Page<E>` or `CursoredPage<E>` | For use with pagination
 |===
 
+[WARNING]
+====
+When working with NoSQL databases, the `count` operation may not be supported, depending on the database structure and capabilities:
+
+- **Key-Value Databases**: In general, the majority of Key-Value databases do not support counting. Their design focuses on retrieving values by keys rather than aggregating data.
+
+- **Other NoSQL Databases**: Support for the `count` operation varies significantly:
+- **Wide-Column Databases**: Counting might require additional configurations, such as enabling secondary indexes or performing costly full table scans, which can affect performance.
```

Comment:
```suggestion
- **Wide-Column Databases**: Support for count is not required. For databases that are capable of returning a count, it might require additional configuration, such as enabling secondary indexes or performing costly full table scans, which can affect performance.
```